test/JDBC/input/BABEL_SCHEMATA-vu-prepare.sql

```diff
@@ -0,0 +1,22 @@
+-- [BABEL-2996]Testing Cross reference schema
+create database db_10;
```

Comment:
this looks like very simple test case for supporting a new view, please sync with Harsh/Shalini to find more variation to cover.

Reply:
Added check that after revoking the usage permission/ownership from user whether it is still showing in catalog or not.